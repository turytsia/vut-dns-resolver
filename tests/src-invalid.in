-r -t -s -x www.fit.vutbr.cz