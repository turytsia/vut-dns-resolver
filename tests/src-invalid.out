Error: Name or service not known
